examples/asr/speech_multitask/speech_to_text_aed.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+# Training the model
+```sh
+python speech_to_text_aed.py \
+    # (Optional: --config-path=<path to dir of configs> --config-name=<name of config without .yaml>) \
+    model.train_ds.audio.tarred_audio_filepaths=<path to tar files with audio> \
+    model.train_ds.audio_manifest_filepath=<path to audio data manifest> \
+    model.validation_ds.manifest_filepath=<path to validation manifest> \
+    model.test_ds.manifest_filepath=<path to test manifest> \
+    model.tokenizer.dir=<path to directory of tokenizer (not full path to the vocab file!)> \
+    model.tokenizer.model_path=<path to speech tokenizer model> \
+    model.tokenizer.type=<either bpe, wpe, or yttm> \
+    trainer.devices=-1 \
+    trainer.accelerator="ddp" \
+    trainer.max_epochs=100 \
```

Comment:
```suggestion
    trainer.max_steps=100000 \
```

Reply:
I'd also add the following options here (or consider adding them to the config):

```
+trainer.limit_train_batches=20000
trainer.val_check_interval=5000
+trainer.use_distributed_sampler=false
```

Reply:
Why train limit batches?

Reply:
comment from @pzelasko: without limit_train_batches code flow triggers `len(dataset)`.  when using lhotse, due to dynamic batch sizes, you're sampling for a total amount of duration, the `len(dataset)` is different for every random seed and not known up front.